cli/Cargo.toml

```diff
@@ -68,11 +70,7 @@ experimental = [
 circuit = ["reqwest", "serde_json", "splinter/sawtooth-signing-compat", "dirs"]
 circuit-auth-type = ["circuit"]
 circuit-template = ["splinter/circuit-template", "circuit"]
-database-migrate-biome = [
-    "splinter/biome-credentials",
-    "splinter/biome-key-management",
```

Comment:
This change should be in a separate commit from the stabilization commit, with some explanation in the commit message about this change.